packages/component-login/client/index.js

```diff
@@ -13,7 +13,7 @@ import NativeLink from 'ui/atoms/NativeLink'
 import ImageWrapper from 'ui/atoms/ImageWrapper'
 import FooterPrivacy from 'ui/atoms/FooterPrivacy'
 import media from 'global/layout/media'
-import { TwoColumnLayout } from '../../global'
+import { TwoColumnLayout } from '../../../app/components/global'
```

Comment:
I think this should be accessible through the `global` alias, may be worth updating this to make it easier to track down when we create the `component-elife-ui` package